src/index.js

```diff
@@ -0,0 +1,14 @@
+import './style.css';
+import displayTodos from './frontend.js';
+import { todos } from './data.js';
+
+const todoapp = () => {
+  let todoLs = JSON.parse(localStorage.getItem('todos'));
+
+  if (todoLs === null) {
+    todoLs = todos;
+  }
+  displayTodos(todos);
```

Comment:
- [ ] As seen in the original design please add the `three-dots` icon and the `enter` icon.

| Original | Yours | 
| ----- | ----- |
| <img src='https://user-images.githubusercontent.com/33323077/126612058-8c2f5dc8-8ab6-4e14-a3ae-0169d5cab793.png' width='250px'> | <img src='https://user-images.githubusercontent.com/33323077/126612167-b2f0b1c7-a744-473a-a41b-5fdf2c40b02d.png' width='250px'> |